CONTRIBUTING.md

```diff
@@ -179,16 +179,16 @@ Please follow all style guides to the best of your abilities.
 
 ### Git commit messages
 
-For all commit messages, please use a short phrase that describes the specific change.
-For example, “Add feature to check normalization method string” is much preferred to “change code”.
-When appropriate, reference issues (via `#` plus number) .
+Pycytominer uses [Conventional Commits](https://www.conventionalcommits.org/en/v1.0.0/) standard for commit messages to aid in automatic changelog generation.
+The easiest way to prepare commit messages that follow this standard is to use [commitizen](https://commitizen-tools.github.io/commitizen/), which comes pre-installed in the devcontainer.
```

Comment:
Minor suggestion to move away from subjective language which might change / have multiple perspectives.
```suggestion
We prepare commit messages that follow this standard using [commitizen](https://commitizen-tools.github.io/commitizen/), which comes pre-installed in the devcontainer.
```

Reply:
Accepted